http/basic/src/main/java/org/wildfly/security/http/basic/BasicAuthenticationMechanism.java

```diff
@@ -134,6 +137,10 @@ public void evaluateRequest(final HttpServerRequest request) throws HttpAuthenti
         }
 
         List<String> authorizationValues = request.getRequestHeaderValues(AUTHORIZATION);
+        if (silent && authorizationValues == null) {
+            log.trace("BASIC authentication mechanism ignored - configuration is set to silent and request does not contain Authorization header");
```

Comment:
@vijay-boddupalli just a minor, we usually add messages to a ElytronMessages file. Like here: https://github.com/wildfly-security/wildfly-elytron/blob/1.x/mechanism/base/src/main/java/org/wildfly/security/mechanism/_private/ElytronMessages.java so that we have messages stored at one place

Reply:
@vijay-boddupalli or actually as Farah pointed out, since this is just a trace message, you do not have to put it in the ElytronMessages class. You do not have to move it.

Reply:
No worries :)